### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by liana-lim**

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React from 'react';
 import Routes from 'src/routes';
 
+
```
Comment (contrapunctus-xv) [inline src/app.js]: not blocking but not necessary

File: src/components/home-page/home-page.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  width: 70%;
+  top: 0;
+  left: 0;
+  border: 15px solid;
+  border-image: url('http://www.designbolts.com/wp-content/uploads/2013/02/Grunge-Seamless-Pattern-For-Website-Background.jpg') 25;
+}
+
+
+//
+//.grey-border {
```
Comment (contrapunctus-xv) [inline src/components/home-page/home-page.scss]: remove

File: src/components/home-page/home-page.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+//  transform: rotate(90deg);
+//  top: 0;
+//  left: 0;
+//  margin-top: -14.75%;
+//  margin-right: auto;
+//  margin-left: 27.5%;
+//  width: 45%;
+//  height: auto;
+//  z-index: -1;
+//}
```
Comment (contrapunctus-xv) [inline src/components/home-page/home-page.scss]: add new line

File: src/components/index.js [hunk 4]
```diff
@@ -1,2 +1,6 @@
 export { default as HomePage } from './home-page/home-page';
 export { default as Navigation } from './navigation/navigation';
+export { default as ContactUs } from './contact-us/contact-us';
+export { default as FAQ } from './faq/faq';
+export { default as OurStory } from './our-story/our-story';
+export { default as WeddingDayInfo } from './wedding-day-info/wedding-day-info';
```
Comment (contrapunctus-xv) [inline src/components/index.js]: new line

File: src/components/navigation/navigation.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  .footer-text-two {
+    text-align: center;
+    vertical-align: middle;
+    padding-top: 25px;
+    position: block;
+    float: right;
+    padding-right: 40px;
+    margin-top: -5px;
+  }
+}
```
Comment (contrapunctus-xv) [inline src/components/navigation/navigation.scss]: new line

File: src/components/navigation/navigation.scss [hunk 6]
```diff
@@ -0,0 +1,41 @@
+.navbar-default {
+  border-color: #FFFFB2;
```
Comment (contrapunctus-xv) [inline src/components/navigation/navigation.scss]: make this a variable

File: src/components/navigation/navigation.scss [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  text-align: center;
+  font-size: 15px;
+  font-family: "Overlock", cursive;
+}
+
+.navbar-brand {
+  border-color: #FFFFB2;
```
Comment (contrapunctus-xv) [inline src/components/navigation/navigation.scss]: use variable

File: src/components/our-story/our-story.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+#ourstory {
+
+}
```
Comment (contrapunctus-xv) [inline src/components/our-story/our-story.scss]: new line

File: src/styles/common.scss [hunk 9]
```diff
@@ -1,6 +1,10 @@
...
-  border-radius:40px;
+  padding-top: 50px;
+  margin: auto;
+  background-color: #FEFDFB;
 }
+
+.container .header {
+  color: black;
+  text-align: center;
+}
```
Comment (contrapunctus-xv) [inline src/styles/common.scss]: new line

File: src/components/home-page/home-page.scss [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+  border-radius: 20px;
+  border-image-source: url('https://s-media-cache-ak0.pinimg.com/originals/5a/12/9e/5a129e73360e54751a8e0c22f9a47a2d.jpg');
+  display: block;
+  margin: auto;
+  height: 25%;
+  width: 25%;
+}
+
+.iceland {
+  background-color: #FEFDFB;
```
Comment (contrapunctus-xv) [inline src/components/home-page/home-page.scss]: make variable

File: src/styles/common.scss [hunk 11]
```diff
@@ -1,6 +1,10 @@
 body {
-  padding-top:50px;
-}
-.container .jumbotron {
-  border-radius:40px;
+  padding-top: 50px;
+  margin: auto;
+  background-color: #FEFDFB;
```
Comment (contrapunctus-xv) [inline src/styles/common.scss]: make variable

**Review by contrapunctus-xv — CHANGES_REQUESTED** [review]

File: src/styles/variables.scss [hunk 12]
```diff
@@ -0,0 +1,4 @@
+$darkslategray: #2f4f4f;
+$ivory: #FEFDFB;
+$lightyellow: #FFFFB2;
+$lightpink: #FFB6C1;
```
Comment (contrapunctus-xv) [inline src/styles/variables.scss]: new line

Comment (liana-lim) [reply src/styles/variables.scss]: -_-;;

**Review by contrapunctus-xv — APPROVED** [review]

**Pull request merged**